src/main/groovy/com/bmuschko/gradle/cargo/tasks/local/CargoConfigureLocal.groovy

```diff
@@ -0,0 +1,8 @@
+package com.bmuschko.gradle.cargo.tasks.local
+
+class CargoConfigureLocal extends LocalCargoContainerTask {
+	CargoConfigureLocal() {
+		action = 'configure'
+		description = 'Configures the container'
```

Comment:
Should say "Configures the local container". Please also add a dot at the end of the description so it aligns with the other task descriptions.